class.jetpack.php

```diff
@@ -5465,17 +5465,22 @@ public function reset_saved_auth_state() {
 		$this->rest_authentication_status = null;
 	}
 
-	function verify_xml_rpc_signature() {
+	function verify_xml_rpc_signature( $token = null, $param_signature = null, $timestamp = null, $nonce = null ) {
```

Comment:
@jeherve Could you help tag some folks here that are well-versed in JP's use of XML-RPC, for review and hints for testing? This is quite a big change to `verify_xml_rpc_signature`, so more testing and insight would be much appreciated <3

Reply:
@mdawaffe would most likely be the best person to ask about this. Not much has changed with this function since the initial release.

Reply:
Going another direction, based on further feedback 👍